tests/system/test_cluster/test_agent_enrollment/test_agent_enrollment.py

```diff
@@ -34,10 +34,10 @@ def clean_environment():
 def test_agent_enrollment(clean_environment):
     """Check agent enrollment process works as expected. An agent pointing to a worker should be able to register itself
     into the master by starting Wazuh-agent process."""
-    # Clean ossec.log and cluster.log
-    host_manager.clear_file(host='wazuh-master', file_path=os.path.join(WAZUH_LOGS_PATH, 'ossec.log'))
-    host_manager.clear_file(host='wazuh-worker1', file_path=os.path.join(WAZUH_LOGS_PATH, 'ossec.log'))
-    host_manager.clear_file(host='wazuh-agent1', file_path=os.path.join(WAZUH_LOGS_PATH, 'ossec.log'))
+    # Clean wazuh.log and cluster.log
+    host_manager.clear_file(host='wazuh-master', file_path=os.path.join(WAZUH_LOGS_PATH, 'wazuh.log'))
+    host_manager.clear_file(host='wazuh-worker1', file_path=os.path.join(WAZUH_LOGS_PATH, 'wazuh.log'))
+    host_manager.clear_file(host='wazuh-agent1', file_path=os.path.join(WAZUH_LOGS_PATH, 'wazuh.log'))
```

Comment:
You can use here the `LOG_FILE_PATH` variable

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/commit/97380191165ffd7168ca42c7a8f98b6e9c177883